pkg/utils/nodepool/nodepool.go

```diff
@@ -55,20 +55,18 @@ func New(provisioner *v1alpha5.Provisioner) *v1beta1.NodePool {
 		},
 		IsProvisioner: true,
 	}
-	if provisioner.Spec.TTLSecondsAfterEmpty != nil {
-		np.Spec.Deprovisioning.EmptinessTTL = &metav1.Duration{Duration: lo.Must(time.ParseDuration(fmt.Sprintf("%ds", lo.FromPtr[int64](provisioner.Spec.TTLSecondsAfterEmpty))))}
-	}
 	if provisioner.Spec.TTLSecondsUntilExpired != nil {
 		np.Spec.Deprovisioning.ExpirationTTL = metav1.Duration{Duration: lo.Must(time.ParseDuration(fmt.Sprintf("%ds", lo.FromPtr[int64](provisioner.Spec.TTLSecondsUntilExpired))))}
 	} else {
 		np.Spec.Deprovisioning.ExpirationTTL = metav1.Duration{Duration: -1}
 	}
-	if provisioner.Spec.Consolidation != nil {
+	if provisioner.Spec.Consolidation != nil && lo.FromPtr(provisioner.Spec.Consolidation.Enabled) {
 		np.Spec.Deprovisioning.ConsolidationPolicy = v1beta1.ConsolidationPolicyWhenUnderutilized
-		np.Spec.Deprovisioning.ConsolidationTTL = metav1.Duration{Duration: lo.Must(time.ParseDuration("15s"))}
```

Comment:
I think we still need to discuss this right?

Reply:
We do, but we can make this change after this PR if the thinking changes. For now, it's just conversion logic which means that it can be handled any way that we want.